packages/kyt-cli/cli/actions/setup.js

```diff
@@ -414,10 +414,25 @@ module.exports = (flags, args) => {
     removeTmpRepo();
```

Comment:
minor but can we rename variables to not refer about "repos"? kind of confusing now since we won't use repos exclusively to manage starter-kyts after this